Code/unavoids.py

```diff
@@ -194,7 +194,7 @@ def getBetasHist(NCDFs_L, BetaSorted, index):
            edges = np.append(edges, [1.01])
            edges[0] = 0.0
 
-        observed_bins = np.where(BetaSorted[:,col] > edges[1], np.minimum(((BetaSorted[:,col]-edges[1])/step).astype('int') + 1, len(edges)-1), 0)
+        observed_bins = np.where(BetaSorted[:,col] > edges[1], np.minimum(((BetaSorted[:,col]-edges[1])/step).astype('int') + 1, len(edges)-2), 0)
```

Comment:
`min(... , len(edges)-1)`  was included just to ensure if some numerical issue with floating points caused an intercept to be slightly above 1.0 (e.g. 1.000000000000001) then it'd still be placed in the last bin and not one past the last bin (causing an error on line 200). I made the change since `len(edges)-2` is the last bin, and the original code actually allowed intercepts to be placed one past the last bin. Therefore we can be sure this case never occurred in our experiments otherwise an error would have been thrown.